<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenLIFUCloudSync</class>
 <widget class="qMRMLWidget" name="OpenLIFUCloudSync">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   
   <item>
    <layout class="QHBoxLayout" name="navLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>‚Üê Back</string>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>

   <item>
    <widget class="ctkCollapsibleButton" name="statusCollapsibleButton">
     <property name="text">
      <string>Cloud Synchronization</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Account Status:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Not Logged In</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Last Sync:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lastSyncLabel">
        <property name="text">
         <string>Never</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>Login to Cloud</string>
     </property>
    </widget>
   </item>
   
   <item>
    <widget class="QPushButton" name="syncButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Sync Now</string>
     </property>
    </widget>
   </item>

   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>

  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>